README.md

```diff
@@ -41,10 +41,21 @@ To run a single test class, run something like this, `gradle test -Dtest.single=
 
 To run tests and compute coverage reports, run `gradle jacocoTestReport`. The report is then in `build/reports/jacoco/test/html/index.html`. (IntelliJ 14 has a good coverage tool that is preferable for development).
 
-To run the main program, run `build/install/gatk/bin/gatk`.
 
 Note: for faster gradle operations, add `org.gradle.daemon=true` to your `~/.gradle/gradle.properties` file.  This will keep a gradle daemon running in the background and avoid the ~6s gradle start up time on every command.  
 
+
+Running GATK 4
+------------
+To run the main program locally, run `build/install/gatk/bin/gatk`.
+
+For spark tools, first build the Spark jar using `gradle sparkJar`. There are three main ways to run gatk 4 with Spark,
```

Comment:
mention where the generated sparkJar will be (in build/libs/) and it's name (something-SNAPSHOT-spark.jar)